### ISSUE TITLE
Prometheus metrics support?

### ISSUE THREAD

**Issue by omerlh** [snyk/broker#117]
How can we monitor the broker? We are planning to host it on Kuberentes, so prometheus metrics support (disabled by default, with paramter that enable them) will be great. This [express middleware](https://github.com/jochen-schweizer/express-prom-bundle) looks promising.

**Comment by adrukh** [comment]
The broker client responds to the `/healthcheck` endpoint, which can be used for uptime monitoring. The path of the health check is configurable with the `$BROKER_HEALTHCHECK_PATH` env var, so you can set it to match your LB standards etc.

This endpoint does not export any metrics though. What kind of metrics would you prefer to see when monitoring the broker client?

**Comment by omerlh** [comment]
You are right, the question we need to answer is what we want to monitor - so I'll refine my issue: What do you think we should be monitoring? Just check the `/health` endpoint? Is there any other important metric? If so, it would be nice to export them via prometheus

**Comment by adrukh** [comment]
Ah we could also report the number of relayed requests (in both directions). This can be a nice future addition.

Right now we find the healthcheck to be more than enough.

**Comment by omerlh** [comment]
Ok, I rely on you to tell us what exactly what we need to monitor in order to make sure broker is working :)

**Comment by mukund-sarma-ck** [comment]
I have the same request. @omerlh did anyone get back to you on this?

**Comment by omerlh** [comment]
No...

**Comment by AlekKras** [comment]
@omerlh any update on this? I have the same request.
